.github/workflows/lint.yml

```diff
@@ -54,6 +54,6 @@ jobs:
       # TODO: Fail on warnings
```

Comment:
```suggestion
```
We can remove this line as a part for this PR.

Reply:
Great, I'll do that!